browserlist.config.js

```diff
@@ -1,7 +1,6 @@
 module.exports = [
-  // TODO: https://jamie.build/last-2-versions
-  'last 2 versions',
-  'ie >= 11',
-  'firefox ESR',
+  '>0.25%',
+  'not ie 11',
```

Comment:
```suggestion
  'not ie all',
```

?

Reply:
Not valid
https://browserslist.dev/?q=ICA%2BMC4yNSUsICAgbm90IGllIDExLCAgIG5vdCBvcF9taW5pIGFsbCwgIG5vdCBkZWFk

Reply:
Okay, just wanted to be sure that NO IE is ever in it again